utils/frame/try-runtime/cli/Cargo.toml

```diff
@@ -18,7 +18,6 @@ log = "0.4.16"
 parity-scale-codec = "3.0.0"
 serde = "1.0.136"
 zstd = { version = "0.10.0", default-features = false }
-
```

Comment:
Why remove these empty lines? They act as some sort of separation between different categories of dependencies.